x/sqlbuilder/marker.go

```diff
@@ -72,3 +73,14 @@ func columnName(m Marker) string {
 
 	return m.ToSQL()
 }
+
+func SQLFunction(m Marker, fn string, args ...string) Marker {
+	return SQLExpression(
+		m.Binding(),
+		fmt.Sprintf(
+			"%s(%s)",
+			fn,
+			strings.Join(append([]string{m.ToSQL()}, args...), ","),
```

Comment:
out of curiosity to you have example SQL function who will use this kind of arguments ?

Reply:
concat ? :D